implementation.md

```diff
@@ -95,7 +95,7 @@ interlocking type systems:
 Shesmu's type system describe the types of every type that can be used in an
 olive. Because `Type` is used by the ASM library, Shesmu's types are
 represented by `Imyhat` objects (this is an Ancient Egyptian word for _mould of
```

Comment:
Not part of the PR, but should this be "mode of conduct"?

Reply:
Dunno. It was what the dictionary had. :man_shrugging: